integ/components/deadline/deadline_04_workerFleetHttps/bin/deadline_04_workerFleetHttps.ts

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
I'm hesitant to do any refactoring to remove duplicate code at this time. 
* There are differences in the requirement for certificates that would need to be added into the HTTPS app (I had left these in my original PR but my next update will remove them from the HTTP app).
* We've got 2 copies, the rule of 3 would say to wait for another file that requires similar code. When we make these run in parallel, it might make sense to split the platforms out into their own CDK apps too to minimize the run time. I'd rather look at how to refactor once at that point than doing it now and then again later.
* It's also unclear to me if these two apps will always need to be kept in sync. If changes don't need to be applied to both of them, refactoring out the duplicate code would just make more difficult.

I don't have very strong feelings against doing this, so if you still think it's a good idea I'll do the refactoring now.

Reply:
These tests will rarely require modifications so long as our API remains stable and since you mentioned we plan to revisit this code soon for parallel execution, we can defer refactoring until then. Resolved.